pygyw/layout/drawings.py

```diff
@@ -98,7 +98,7 @@ def __init__(self,
         :type left: int
         :param left: The vertical offset. Defaults to 0.
         :type top: int
-        :param font: The font used for the text. Defaults to None.
+        :param font: The font used for the text.
```

Comment:
Can you specify the value in the parameter documentation, like it is done with `left` pls ?